src/components/Layout/Footer.tsx

```diff
@@ -1,69 +1,118 @@
-import styled from 'styled-components';
-import Link from 'next/link'
-import { Color, Media } from '@src/const/styles/variables'
-import { scrollToClickedElement } from '@src/lib/analytics/events';
+import styled from "styled-components";
+import Link from "next/link";
+import { Color, Media, Font } from "@src/const/styles/variables";
+import { Section } from "@src/const/styles/pages";
+import { Logo } from "./Header";
```

Comment:
Nitpick: It's pretty weird, that `Footer` depends on `Header`. Maybe make it sense to move `Logo` to another file? Like a `commonComponents.tsx`

Reply:
Moved to `components/common/Logo`